benchmark/python/benchmarks/resnet_benchmark.py

```diff
@@ -0,0 +1,43 @@
+import torch
+import numpy as np
+from mpact.models.resnet import resnet20
+from mpact_benchmark.utils.benchmark_utils import benchmark, Backends
+
+
+@benchmark(
+    [
+        {
+            "name": f"{fmt}_{shape}_{dtype.__name__}",
+            "shape": shape,
+            "formats": fmt,
+            "dtype": dtype,
+            "drange": (1, 100),
+            "sparsity": [0.5, 0.9],
+            # Torch inductor has slightly different results.
```

Comment:
slightly different, as in beyond torch.allclose()?

can we change some of the parameters (within reason)